### ISSUE TITLE
Not showing as Contributor

### ISSUE THREAD

**Issue by MiKr13** [FaisalUmair/udemy-downloader-gui#318]
Hi,

Few months ago I created a pull request: [291](https://github.com/FaisalUmair/udemy-downloader-gui/pull/291), this has been merged into master and it included:

- Open Course in browser on Button/Thumbnail click.
- Beautification of code.

I haven't been added as a contributor though, I think this is a confusion, please look into it, working on some other requested features too meanwhile.

**Comment by MiKr13** [comment]
That's great to hear man, motivating also, yeah I was just going through issues which have newer demands in it and was implementing one of the features which asked for saving data in one folder, plus I also wanted to split the code into smaller files so that it's easier to maintain if you don't mind splitting.

Thanks again and looking forward to making this a great tool for millions of Udemy users.

**Comment by FaisalUmair** [comment]
@MiKr13  I don't mind any kind of help at all. I am also working on v2, for which I had planned to use VueJS which will add some modularity. But if you are willing to split the code and use them as modules, I would say go ahead because v2 will still take some time, as I only work on it in my free time.

And also feel free to make any sort of contributions, I will review them and merge, but please make sure you test the app properly, after making any changes,  I don't have automatic testing setup.

**Comment by MiKr13** [comment]
@FaisalUmair I'll do stuff responsibly.

**Comment by MiKr13** [comment]
@FaisalUmair are those changes integrated now?

**Comment by FaisalUmair** [comment]
Hi @MiKr13  Not yet.  I can merge that now that's not an issue, but I want to do it when I am ready to work on it myself as well to make the changes. I am currently busy with some projects and  once I have some free time I will take your changes for sure.

**Comment by MiKr13** [comment]
@FaisalUmair That's great to hear, I was just curious as our Company gives credits on Open Source Contributions made in a quarter, I am new there so am trying to get more into Open Source and get better projects there, that's why. It's great to still hear from you that you have that in mind, Thanks.

**Comment by FaisalUmair** [comment]
Yes I could assume that earlier also, since you were concerned about your contribution not showing up.  But no worries, I know you did your work, so I will merge it soon (when I work on it) and then make the changes which I want.

**Comment by MiKr13** [comment]
Thanks for understanding, keep up the great work.

**Comment by MiKr13** [comment]
@FaisalUmair Hi, I just wanted to know when will you accept me as a contributor?

**Comment by MiKr13** [comment]
Hi @FaisalUmair are you still going to merge it or should I close this?

**Comment by FaisalUmair** [comment]
Hi @MiKr13 I am currently working on 2.0 branch and not focused on master.

But let's do one thing, please fork the repo again, and implement the same thing, make a new pull request and I will merge it within 2-3 days from the time you make a pull request.

**Comment by MiKr13** [comment]
Hi @FaisalUmair I am looking into 2.0 and was looking at some requests too actively. I will be contributing soon there.

I have created one pull request again as you asked: #488

**Comment by MiKr13** [comment]
Hi, @FaisalUmair One thing I wondered, why you switched from Vue to React?

**Comment by FaisalUmair** [comment]
HI @MiKr13 I switched to React because over the last year, I have worked with projects involving React so I am more comfortable using React than Vue.

And for now, I might not be accepting any pull request for 2.0. As it is incomplete and I might change a lot of things frequently. Once it is complete, I will accept pull requests to it.
